tests/test_skeleton.py

```diff
@@ -0,0 +1,157 @@
+        nclicks,
+        prop_name,
+        prop_value,
+        prop_type=None
+):
+    """This function is the code of a callback which is triggered by
+    the button on the simple app used in the test.
+    :param nclicks (int): The n_clicks value of the button in the 
+                          simple app
+    :param prop_name (string): The name of the property that is to be
+                               modified
+    :param prop_value (string): The value that is to be assigned to the
+                                prop defined by prop_name.
+    :prop_type (string): One of the predefined types in PROP_TYPES. 
+    :return: The value that is to be assigned to the prop defined by 
+             prop_name, after casting it to the correct type. 
+    """
+
+    # avoid triggering this callback when the button is first created
+    if nclicks is not None:
+        # cast the string representation of the desired prop value
+        # into the appropriate type
+
+        if prop_type in PROP_TYPES:
+            return PROP_TYPES[prop_type](prop_value)
+
+        # define any other "translations" from a string to a datatype
+        # below; e.g.,
+        # elif prop_type == 'list': return ...
+
+    return None
+
+
+# below, write tests for changing the props of the React component
+# (following this basic structure) 
+
+@init_demo_app(APP_NAME)
+def test_PROPNAME(dash_threaded, selenium):
+    """Test that some prop updates correctly when changed."""
+    
+    def assert_callback(
+            component_PROPNAME,
+            nclicks,
+            input_PROPNAME
+    ):
+        """Determines the pass/fail status of this test.
+
+        :param (string) component_PROPNAME: The value of PROPNAME for the 
+                                            component after it is set
+        :param (int) nclicks: The number of clicks on the button in the
+                              simple test app (not used here) 
+        :param (string) input_PROPNAME: The value of PROPNAME that is sent
+                                        to the component. 
+        
+        :return (string): 'PASSED' for a test that passed, or 'FAILED'
+                          for a test that failed
+        """
+        # avoid triggering this callback when the button is first created
+        if nclicks is not None:
+            
+            # check for the pass/fail condition here; this is a
+            # shallow comparison, so write your own if necessary
+            if component_PROPNAME == input_PROPNAME:
+                return PASS
+
+        return FAIL
+
+    # replace "None" with a string that defines the type of the prop
+    # (e.g., 'int', 'float', 'list')
+    propType = None  
+
+    template_test_component_single_prop(
+        dash_threaded,
+        selenium,
+        APP_NAME,
+        assert_callback,
+        COMPONENTNAME_test_props_callback,
+        PROPNAME,
+        input_PROPNAME,
+        propType=propType,
```

Comment:
it should be `prop_type` in snake case ;)

Reply:
Fixed in 709f8eb!